.github/workflows/ci.yml

```diff
@@ -8,14 +8,18 @@ jobs:
     runs-on: ${{matrix.os}}
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
       - uses: dcodeIO/setup-node-nvm@master
         with:
           node-version: ${{matrix.node}}
-      - run: npm install -g yarn && yarn --frozen-lockfile
-        env:
-          CI: 'true'
-      - name: Test
-        run: yarn test
+      - uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: npm-${{hashFiles('package-lock.json')}}
+          restore-keys: npm-
```

Comment:
if the setup-node action is used, this could be abstracted away https://github.blog/changelog/2021-09-07-github-actions-setup-node-supports-dependency-caching-for-projects-with-monorepo-and-pnpm-package-manager/

Reply:
I’m assuming the `package-lock.json` will change a ton (it’s 1.4mb, so it’s likely something changes in the ecosystem every day), which is why I’m investigating a different cache strategy.
See also https://github.com/bahmutov/npm-install/issues/108.

Oh, I’m just seeing that `setup node` finally supports “nvm lts syntax” (`lts/erbium`), then we can at least switch to that